src/app/components/AssetSelector/index.jsx

```diff
@@ -112,11 +112,13 @@ AssetSelector.propTypes = {
   selectAsset: PropTypes.func.isRequired,
   supportedAssetSymbols: PropTypes.arrayOf(PropTypes.string),
   portfolioSymbols: PropTypes.arrayOf(PropTypes.string),
+  handleDisabledAssets: PropTypes.func
 }
 
 AssetSelector.defaultProps = {
   supportedAssetSymbols: [],
   portfolioSymbols: [],
+  handleDisabledAssets: (asset) => !asset.swapEnabled
```

Comment:
It doesn't really make sense to call this a handler. Generally a `handle` function responds to an event or action like `onClick`. Since this function is a predicate the prefix `is` makes more sense. Could you rename it to `isAssetDisabled`?